### ISSUE TITLE
Add instructions / requirements for better user input handling.

### ISSUE THREAD

**Issue by joethe** [UMM-CSci-3601/3601-lab2_client-server#23]
I feel like there's a good opportunity in this lab to talk about gracefully handling user input. We should add some basic instructions / requirements along the lines of handling things like case sensitivity in text fields,  invalid requests, etc.

**Comment by NicMcPhee** [comment]
@kklamberty – the form validation work you've done on Lab 4 (and which should move back to Lab 3) seems to address this to some degree. Do you think that we want/need this here, or is having it come up later OK?

**Comment by pluck011** [comment]
@NicMcPhee - @koran023 , @kklamberty , and I talked about that this term. Perhaps we will include some validation for user input on filters in lab 3. https://github.com/UMM-CSci-3601/3601-lab3_angular-spark/issues/190 . We are still discussing if it would make sense to worry about handling odd data like negative ages since that won't happen except for thing like (maybe) URL hacking once the validation is in place in later work.

**Comment by helloworld12321** [comment]
@NicMcPhee do you think we should we close this old issue as `wontfix`?

It seems like it's not something we're planning on adding to this lab. (And for my part, I kind of like the idea of keeping this lab simple/less feature-rich! 🙂)

**Comment by NicMcPhee** [comment]
@helloworld12321 – I have mixed feelings on this. I agree that complicating the labs isn't helpful. On the other it would be a good opportunity to introduce some error handling ideas since that's an area where folks often don't do a great job?

**Comment by helloworld12321** [comment]
Ah, sounds good! In that case I'm happy to leave it. :)

**Comment by kklamberty** [comment]
Just as a note, I did talk a bit in class today about making sensible choices for input that prevent users from searching from things in ways that won't know what they mean (like, freeform typing where you might spell things wrong compared to selecting from a set of feasible options).
